docs/tasks/run-application/force-delete-stateful-set-pod.md

```diff
@@ -5,7 +5,7 @@ approvers:
 - foxish
 - smarterclayton
 title: Force Delete StatefulSet Pods
-redirect:from:
+redirect_from:
```

Comment:
Lines 8-10 should be removed since we are now using the _redirects file for server-side redirects.